### DISCUSSION TITLE
Some added types lost

### DISCUSSION THREAD

**Discussion by na-nederland** [anchovie91471/12-Step-Meeting-List-Xtras#11] [General]
We would suggest a backup/restore export/import function for 12 Step Meeting List Xtras settings and manual added types.

**Comment by anchovie91471** [comment]
Custom types are stored in the database so some of all of the settings must've gotten erased. I'll take a look to see if there's anything in the plugin that might do that (like disabling it).

I'm sorry this happened. I'll definitely consider adding a backup feature. That shouldn't be too difficult.

**Reply by na-nederland** [reply]
Thanks and much appreciated!
